api/src/main/java/com/alibaba/nacos/api/common/Constants.java

```diff
@@ -42,6 +44,12 @@
 
     public static final String GROUP = "group";
 
+    public static final String HEART_BEAT_TIMEOUT = "heartBeatTimeout";
```

Comment:
Please define these keys in class PreservedMetadataKeys and the keys should start with 'preserved.'

Reply:
ok